### DISCUSSION TITLE
SHUTDOWN error at a step

### DISCUSSION THREAD

**Discussion by gregma** [bassamanator/Sovol-SV06-firmware#131] [General]
I got to the section "Configure your printer"

--Do a mechanical gantry calibration; G34. After the controlled collision against the beam at the top, there will be a 10 second pause for you to verify that both sides of the gantry are pressed up against the stoppers at the top.

The printer went close to the top beam, but then threw a Lost communication with the MCU 'mcu'.

Pi4b, MainsailOS fully updated.  All steps followed.  Guide says to stop on any errors, so I'm stopped at that point.

FYI:  Neither RESTART nor FIRMWARE RESTART do anything.

gregma

**Comment by bassamanator** [comment]
Check your usb cable.
